cmd/geth/main.go

```diff
@@ -417,7 +417,7 @@ func startNode(ctx *cli.Context, stack *node.Node, backend ethapi.Backend, isCon
 					continue
 				}
 				if timestamp := time.Unix(int64(done.Latest.Time), 0); time.Since(timestamp) < 10*time.Minute {
-					log.Info("Synchronisation completed", "latestnum", done.Latest.Number, "latesthash", done.Latest.Hash(),
```

Comment:
I think this should be Synchronization (or Synchronisation)

Reply:
Please revert this change

Reply:
Got, I have reverted